cpu/kinetis/include/cpu_conf_kinetis.h

```diff
@@ -158,6 +160,28 @@ extern "C"
 #endif /* MK20D7_H_ */
 /** @} */
 
+/**
+ * @name MKE02Z Compatibility layer
+ * @{
+ */
+#if defined(MKE02Z_H_)
+#define PORT_PCR_MUX(n) 0
+#define PORT_PCR_PE_MASK 0
+#define PORT_PCR_PS_MASK 0
+#define PORT_PCR_IRQC(n) 0
+#define ADC_CFG1_MODE(n) 0
+#define SPI_CTAR_CPHA_MASK 0
+#define SPI_CTAR_CPOL_MASK 0
+typedef struct {} LPTMR_Type;
+#define MCG_C4_DRST_DRS(n) 0
+#define MCG_C4_DMX32_MASK 0
+#define MCG_C2_RANGE0(n) 0
+#define PORTA_BASE 0
+#define SIM_SCGC5_PORTA_SHIFT 0
+#define PIT_CLKEN() (bit_set32(&SIM->SCGC, SIM_SCGC_PIT_SHIFT))
```

Comment:
Move PIT_CLKEN to around line 130 and make it conditional on `SIM_SCGC_PIT_SHIFT`:
```c
/** Enable PIT clock gate */
#if defined(SIM_SCGC6_PIT_SHIFT)
/* Most Kinetis models */
#define PIT_CLKEN()    (bit_set32(&SIM->SCGC6, SIM_SCGC6_PIT_SHIFT))
#elif defined(SIM_SCGC_PIT_SHIFT)
/* KE0x family */
#define PIT_CLKEN()    (bit_set32(&SIM->SCGC, SIM_SCGC_PIT_SHIFT))
#endif

Reply:
Done.